package.json

```diff
@@ -165,10 +165,10 @@
     "cozy-flags": "1.5.1",
     "cozy-interapp": "0.2.18",
     "cozy-konnector-libs": "4.12.4",
-    "cozy-pouch-link": "5.4.2",
```

Comment:
oula c'est chelou d'avoir 2 `cozy-pouch-link`

Reply:
I don't know what went wrong. Maybe a fail during a rebase. But since it's now OK, I'm in favor of merging as is.